From b72686b694ac4979450853a6a5dc2ece3df9fe82 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Tue, 27 Jul 2021 11:17:08 +0100
Subject: [PATCH] media: i2c: tc358743: Fix compiler warning

Avoid a compiler warning by using the "fallthrough" pseudo-keyword in
place of the old "/* fall through */" comment convention.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 drivers/media/i2c/tc358743.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/media/i2c/tc358743.c
+++ b/drivers/media/i2c/tc358743.c
@@ -2008,7 +2008,7 @@ static int tc358743_probe_of(struct tc35
 	switch (bps_pr_lane) {
 	default:
 		dev_warn(dev, "untested bps per lane: %u bps\n", bps_pr_lane);
-		/* fall through */
+		fallthrough;
 	case 594000000U:
 		state->pdata.lineinitcnt = 0xe80;
 		state->pdata.lptxtimecnt = 0x003;
